test/test.retry.ts

```diff
@@ -275,4 +275,30 @@ describe('🛸 retry & exponential backoff', () => {
     });
     scope.done();
   });
+
+  it('should delay the initial retry by 100ms by default', async () => {
+    const scope = nock(url).get('/').reply(500).get('/').reply(200, {});
+    const start = Date.now();
+    await request({
+      url,
+      retry: true,
+    });
+    const delay = Date.now() - start;
+    assert.ok(delay > 100 && delay < 150);
+    scope.done();
+  });
+
+  it('should respect the retryDelay if configured', async () => {
+    const scope = nock(url).get('/').reply(500).get('/').reply(200, {});
+    const start = Date.now();
+    await request({
+      url,
+      retryConfig: {
+        retryDelay: 500,
```

Comment:
This seems like a nice simple way to write this test for now 👍 

I've definitely seen tests like this that add 100+ ms delays add up over time, and lead to a fairly slow test suite... I don't think this will bite us, having a couple smoke tests for retry logic.